src/components/textfieldSlider/js/TextfieldSlider.js

```diff
@@ -26,15 +26,19 @@ var fluid_2_0_0 = fluid_2_0_0 || {};
                 container: "{textfieldSlider}.dom.textfield",
                 options: {
                     model: "{textfieldSlider}.model",
-                    range: "{textfieldSlider}.options.range"
+                    range: "{textfieldSlider}.options.range",
+                    ariaOptions: "{fluid.textfieldSlider}.options.ariaOptions",
+                    strings: "{fluid.textfieldSlider}.options.strings"
```

Comment:
I think we should either use `{fluid.textfieldSlider}` or `{textfieldSlider}`. It seems they are referring to the same component, so we should use a consistent way of referring to it.